lib/urn.js

```diff
@@ -31,7 +31,8 @@ module.exports = {
       'pbk:rsa:2048',
       'ble:1\.0',
       'nfc:1\.0',
-      'sn'
+      'sn',
+      'hash:argon'
```

Comment:
@joshbeam, Looks good, though on which level do you want to convert argon's params with hash into hex?
If this will be as input, e.g. `hash:argon:4466aabbeeff`
Than that's fine, for now. 
Also is that string include argon version as well?